context/app/routes_auth.py

```diff
@@ -99,8 +99,12 @@ def login():
         nexus_token=nexus_token,
         is_authenticated=True,
         user_email=user_email)
+
+    previous_path_and_search = unquote(request.cookies.get('urlPathAndSearchBeforeLogin'))
```

Comment:
Is this just the path and search? or the whole url? I can't remember why the `_external=True` was needed below, but should make sure we don't trip over the same issue. (Looking at the `git blame` might refresh my memory, if needed.)